docs/decisions/update_puppeteer_2024_07_28.md

```diff
@@ -0,0 +1,62 @@
+# Use response data to detect sign-in success
+
+## Context and Problem Statement
+
+Should we update to the newest version of puppeteer, [22.15.0](https://github.com/puppeteer/puppeteer/blob/main/packages/puppeteer-core/CHANGELOG.md#22150-2024-07-31)?
```

Comment:
The newest version is now v23, which includes Firefox support! It's a nice thing to have, not urgent though.  https://hacks.mozilla.org/2024/08/puppeteer-support-for-firefox/